src/lib/genesis_ledger_helper/lib/genesis_ledger_helper_lib.ml

```diff
@@ -594,66 +594,6 @@ let make_genesis_constants ~logger ~(default : Genesis_constants.t)
         (config.daemon >>= fun cfg -> cfg.minimum_user_command_fee)
   }
 
-let runtime_config_of_genesis_constants (genesis_constants : Genesis_constants.t)
-    : Runtime_config.Genesis.t =
-  { k = Some genesis_constants.protocol.k
-  ; delta = Some genesis_constants.protocol.delta
-  ; slots_per_epoch = Some genesis_constants.protocol.slots_per_epoch
-  ; slots_per_sub_window = Some genesis_constants.protocol.slots_per_sub_window
-  ; grace_period_slots = Some genesis_constants.protocol.grace_period_slots
-  ; genesis_state_timestamp =
-      Some
-        (Genesis_constants.genesis_timestamp_to_string
-           genesis_constants.protocol.genesis_state_timestamp )
-  }
-
-let runtime_config_of_precomputed_values (precomputed_values : Genesis_proof.t)
```

Comment:
This doesn't seem like what you really want here -- runtime_config is already a field in `precomputed_values`. Furthermore, this was only being used in one place so I just inlined it.

On a related note, I want to open up https://github.com/MinaProtocol/mina/issues/16018 for discussion